csharp/tests/Facebook.Yoga.Universal.Tests/Facebook.Yoga.Universal.Tests.csproj

```diff
@@ -0,0 +1,146 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It was not possible to use the generated tests in the universal test project because those tests require NUnit 2.6.4 which does not provide a UWP NuGet package... A subset of tests has been added and converted to MSTest to allow some tests to run.